docs/sources/setup/install/helm/install-scalable/aws.md

```diff
@@ -0,0 +1,580 @@
+       # admin: "<Insert s3 bucket name>" # Your actual S3 bucket name (loki-aws-dev-admin) - GEL customers only
+     s3:
+       region: <Insert s3 bucket region> # eu-west-2
+       #insecure: false
+     # s3forcepathstyle: false
+       
+       
+ deploymentMode: SimpleScalable
+
+ serviceAccount:
+   create: true
+   annotations:
+     "eks.amazonaws.com/role-arn": "arn:aws:iam::<Account ID>:role/LokiServiceAccountRole" # The service role you created
+
+ backend:
+   replicas: 2
+   persistence:
+     storageClass: gp2
+     accessModes:
+       - ReadWriteOnce
+     size: 10Gi
+ read:
+   replicas: 2
+   persistence:
+     storageClass: gp2
+     accessModes:
+       - ReadWriteOnce
+     size: 10Gi
+ write:
+   replicas: 3
+   persistence:
+     storageClass: gp2
+     accessModes:
+       - ReadWriteOnce
+     size: 10Gi
+
+ # This exposes the Loki gateway so it can be written to and queried externaly
+ gateway:
+   service:
+     type: LoadBalancer
+
+
+ # Enable minio for storage
+ minio:
+   enabled: false
+ 
+```
+
+It is critical to define a valid `values.yaml` file for the Loki deployment. To remove the risk of misconfiguration, let's break down the configuration options to keep in mind when deploying to AWS:
+
+- **Loki Config vs. Values Config:**
+  - The `values.yaml` file contains a section called `loki`, which contains a direct representation of the Loki configuration file.
+  - This section defines the Loki configuration, including the schema, storage, and querier configuration.
+  - The key configuration to focus on for chunks is the `storage_config` section, where you define the S3 bucket region and name. This tells Loki where to store the chunks.
+  - The `ruler` section defines the configuration for the ruler, including the S3 bucket region and name. This tells Loki where to store the alert and recording rules.
+  - For the full Loki configuration, refer to the [Loki Configuration]({{< relref "../../../../configure" >}}) documentation.
+
+- **Storage:**
+  - Defines where the Helm chart stores data.
+  - Set the type to `s3` since we are using Amazon S3.
+  - Configure the bucket names for the chunks and ruler to match the buckets created earlier.
+  - The `s3` section specifies the region of the bucket.
+
+- **Service Account:**
+  - The `serviceAccount` section is used to define the IAM role for the Loki service account.
+  - This is where the IAM role created earlier is linked.
+
+- **Gateway:**
+  - Defines how the Loki gateway will be exposed.
+  - We are using a `LoadBalancer` service type in this configuration.
+
+{{< admonition type="warning" >}}
+Make sure to replace the placeholders with your actual values.
+{{< /admonition >}}
+
+### Deploy Loki
+
+Now that you have created the `values.yaml` file, you can deploy Loki using the Helm chart.
+
+1. Deploy using the newly created `values.yaml` file:
+
+    ```bash
+    helm install --values values.yaml loki grafana/loki -n loki
```

Comment:
worth calling out that `-n loki` is creating the loki namespace to install this in? and that's configurable? Also, `--create-namespace` can be added if the namespace doesn't already exist (which I don't think I saw instructions for creating 🤔 )

Reply:
I had it as a separate step this is way more concise so updated and added a note. I kept it deliberate that they create a Loki namespace as the service account created earlier is only valid for that namespace but that can also be changed